Sources/PointFree/Stripe.swift

```diff
@@ -128,7 +128,7 @@ public struct Stripe {
   public struct Invoice: Codable {
     public private(set) var amountDue: Cents
     public private(set) var amountPaid: Cents
-    public private(set) var charge: Charge?
+    public private(set) var charge: Either<Charge.Id, Charge>?
```

Comment:
This is the root of the bug. When we added support to view historic invoices, we needed to expand information to see the charge. Stripe webhooks do _not_ expand this kind of info, so we need to support the unexpanded state. `Either` and conditional conformance makes this simple!